android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMKeyboard.java

```diff
@@ -325,6 +326,9 @@ public void onConfigurationChanged(Configuration newConfig) {
     loadJavascript(KMString.format("setBannerHeight(%d)", bannerHeight));
     loadJavascript(KMString.format("setOskWidth(%d)", newConfig.screenWidthDp));
     loadJavascript(KMString.format("setOskHeight(%d)", oskHeight));
+
+    RelativeLayout.LayoutParams params = KMManager.getKeyboardLayoutParams();
+    this.setLayoutParams(params);
```

Comment:
Matches #5521.

Reply:
> "is in both"?

Like... both the in-app and the system keyboard?

The reason I thought to put it here:  this is the function that's called whenever the keyboard is rotated.  So, that's the obvious time to actually do code that handles rotation things.  I actually don't know why the black bar wasn't happening in the Android app's system keyboard before this fix, but if I did know, I'd probably knock out the other block of code to confirm that this is sufficient to handle both contexts.  DRY > WET and all that.

If it turns out that this needs tweaks based on the review of #5521, I'll make sure those adjustments are made here too.

Reply:
Went and double-checked; the system keyboard works fine.

Reply:
actually, meant "is in both PRs" -- this one and #5521!

Reply:
Oh.  This is where I first made the fix, then I pseudo-🍒'd it to that PR.

Our tester's been labeling tests as failures when the black bar shows up, despite it being a known separate issue.  So, it'll help prevent false-negatives.  Also... the fact is that between this PR and the next, we're tweaking so many related aspects of the OSK that I need to be _sure_ that this PR doesn't turn around and re-break the fix instantly.